examples/pubsub.memory/go.mod

```diff
@@ -0,0 +1,31 @@
+module github.com/dapr-sandbox/components-go-sdk/examples/pubsub.memory
+
+go 1.19
+
+replace github.com/dapr-sandbox/components-go-sdk => ../../.
```

Comment:
I think we should have a git hook to avoid those "replace" finding their way into commits